bindata/network/ovn-kubernetes/ipsec.yaml

```diff
@@ -182,7 +194,7 @@ spec:
           # Start the pluto IKE daemon and use the ipsec configuration file that
           # persists across container restarts. If the container has been restarted,
           # this will contain the previous configuration from ovs-monitor-ipsec
-          /usr/libexec/ipsec/pluto --leak-detective --config /etc/openvswitch/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
+          /usr/libexec/ipsec/pluto --leak-detective --config /etc/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
```

Comment:
This doesn't match the above comment; this file isn't bind-mounted and thus won't be persisted, right?

Reply:
Ooops. Forgot to update comment.